web/Conventions_for_Languages/Spanish-specific-translations.qmd

```diff
@@ -4,72 +4,97 @@ title: "Spanish"
 
 [![](https://translate.rx.studio/widgets/r-project/es/svg-badge.svg?native=1){fig-align="left" fig-alt="translation status"}](https://translate.rx.studio/languages/es/r-project/)
 
-_This repository is a hub for all people who want to help translate R into Spanish. This README is in English, but you 
-can reach out to other translators on the R Contributors Slack if you need any additional information about the 
-project._
+_This page is a hub for all people who want to help translate R messages into Spanish. You 
+can reach out to other translators on the R Contributors Slack if you need any additional information about the project._
 
 ## About
 
 The goal of the _es_ project is to bring together all people interested in helping translate the [R 
 language](https://en.wikipedia.org/wiki/R_(programming_language)) into Spanish. If you use R a lot, we need your help!
 
-Unlike other languages, R tries to display every message in the same language as the computer it's running on. In 
-principle, this reduces R's barrier to entry; warnings and errors are very common in code written by beginners, so it's 
-essential that these messages are as clear as possible.
+Unlike other languages, R tries to display every message in the same language as the computer it's running on. 
+In principle, this reduces R's barrier to entry; warnings and errors are very common in code written by beginners (or anyone really), so it's essential that these messages are as clear as possible.
 
 ## Resources
 
 ### Style
 
+R is a programming language that's almost 30 years old and, therefore, its messages were written by many people with very different writing styles. 
+When translating, we what to generate a version of the English message the way we usually we will say it in Spanish. 
+In some cases this may involve altering the order of the elements of a sentence or adding words that were not in the original. The important thing is that the translation "sounds" as natural as possible in Spanish, respecting the content that was intended to be conveyed in the original.
+
+We also don't translate anything that is a technical term from R like, for example, function names, objects and arguments (usually between quotes).
+
+The dialectal variety of Spanish that we will use in the translation is the Latin american (because the possible target audience that speaks it is wider). We will try to make it as neutral a version as possible, therefore:
+
+* We will avoid local/regional expressions or usages, i.e., that are not widespread throughout Latin America.
+* We will not use voseo (vos/vosotros).
+
 ### Formatting
 
-### Grammatical gender
+The use of spaces and paragraphs in Portuguese is standardized and well established. R texts, however, have space and 
```

Comment:
I'm not sure I understand the reference to Portuguese here. Why is it added? The guidelines are for Spanish.